nixos/modules/system/boot/networkd.nix

```diff
@@ -131,34 +173,67 @@ let
           "fou"
           "xfrm"
           "ifb"
+          "bareudp"
           "batadv"
         ])
         (assertByteFormat "MTUBytes")
         (assertMacAddress "MACAddress")
       ];
 
+      sectionBridge = checkUnitConfig "Bridge" [
+        (assertOnlyFields [
+          "HelloTimeSec"
+          "MaxAgeSec"
+          "ForwardDelaySec"
+          "AgeingTimeSec"
```

Comment:
do these take only ints, or systemd time strings? from the docs it looks like they're plain ints that could be checked for, haven't tested that though.

Reply:
none of the other `*Sec` Options are checked. I would kinda could imagine that systemd offers support for other time units. As far as i remember they usually only mention the base unit and offer support for others ...

But i haven't checked either.